src/codeu/chat/server/Server.java

```diff
@@ -280,6 +308,7 @@ public void run() {
                     relay.pack(user.id, user.name, user.creation),
                     relay.pack(conversation.id, conversation.title, conversation.creation),
                     relay.pack(message.id, message.content, message.creation));
+  private static final ServerInfo info = new ServerInfo();
```

Comment:
This should be added close to the top of this file, near the other similar declarations of member variables, maybe around line 55 or so?